install/kubernetes/cilium/values.yaml

```diff
@@ -1771,7 +1771,7 @@ envoy:
   terminationGracePeriodSeconds: 1
 
   # -- TCP port for the health API.
-  healthPort: 9879
+  healthPort: 9878
```

Comment:
This needs to be documented in the list of ports that we open on the host.

Reply:
This is done in the second to last or last commit.

Reply:
@aanm 

yep that one should be covered